navigation2/Dockerfile

```diff
@@ -67,10 +67,11 @@ RUN rm -rf src build log
 
 # Install rviz2 to send goals to Nav2. 
 # TODO(xfiderek): Remove rviz2 and humble-nav2 froms this image
-RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
-    --mount=type=cache,target=/var/lib/apt,sharing=locked \  
-    sudo apt update && \
-    sudo apt install -y ros-humble-rviz2 ros-humble-nav2-bringup
+# TODO(asimonov): disabling rviz and bringup apt install after move to jazzy
+#RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+#    --mount=type=cache,target=/var/lib/apt,sharing=locked \  
+#    sudo apt update && \
+#    sudo apt install -y ros-humble-rviz2 ros-humble-nav2-bringup
```

Comment:
@asimonov  Why can't rviz and bringup be installed? Is there another way to bring them into the image?

Reply:
space_nav2_bringup is already merged by: https://github.com/space-ros/docker/pull/152/files

and it is in the docs: https://github.com/space-ros/docker/blob/47d23b1880b8f59d9b3398b82f8a4a27a7660d23/nav2_demo/README.md?plain=1#L33

so we can remove it safely

Reply:
i re-instated rviz2 apt install with comment to remove it in the future.